UnitsNet.Tests/CustomCode/AmountOfSubstanceTests.cs

```diff
@@ -42,6 +42,17 @@ public class AmountOfSubstanceTests : AmountOfSubstanceTestsBase
         protected override double NanopoundMolesInOneMole => 0.002204622621848776 * 1e9;
         protected override double PoundMolesInOneMole => 0.002204622621848776;
         protected override double MegamolesInOneMole => 1e-6;
+        
+        private static double MolarMassOfOxygen = 15.999;
+        private static double MolesInTenGramsOfOxygen = 0.6250390649415588;
+
+
+        private static double MolarMassHClInGramsPerMole = 36.46;
```

Comment:
Reuse the new `KnownQuantities` class?
And still need to inline the test case values here.